plume-models/src/blogs.rs

```diff
@@ -142,7 +142,7 @@ impl Blog {
     }
 
     fn fetch_from_webfinger(conn: &Connection, acct: &str) -> Result<Blog> {
-        resolve(acct.to_owned(), *USE_HTTPS)?
+        resolve(acct.to_owned(), CONFIG.use_https)?
```

Comment:
Not really related, but I'm thinking about it because I'm seeing it: shouldn't we drop this setting and force people to set up HTTPS. I introduced it to make it easier to develop locally, but now this requires HTTPS too because of CSRF.

Reply:
We should probably, http does not work well anyway

Reply:
As someone on Matrix said they were using Plume on a Tor Hidden Service, I tried to see if this won't cause any CSRF issues, and apparently Tor Browser consider hidden service to be secure even without https, that's nice